text/src/autogluon/text/automm/predictor.py

```diff
@@ -1325,36 +1493,17 @@ def save(
                 ensure_ascii=True,
             )
         # In case that users save to a path, which is not the original save_path.
-        if path != self._save_path:
-            shutil.copy(os.path.join(self._save_path, "model.ckpt"), path)
-
+        if os.path.abspath(path) != os.path.abspath(self._save_path):
+            model_path = os.path.join(self._save_path, "model.ckpt")
+            if os.path.isfile(model_path):
+                shutil.copy(model_path, path)
+                
     @staticmethod
-    def load(
-            path: str,
-            resume: Optional[bool] = False,
-            verbosity: Optional[int] = 3,
+    def _load_metadata(
```

Comment:
Instead of initializing a predictor inside this function, we can pass a predictor to it and load the metadata this predictor.